master/static/migrations/20231208132223_drop-raw-checkpoints.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE raw_checkpoints CASCADE;
+
+ALTER TABLE public.checkpoints_v2
```

Comment:
blocking: this feels a little risky to me

we are adding a ```NOT NULL``` constraint to a column that previous was ```NULLABLE``` meaning if some customer somehow had written a ```NULL``` into any checkpoint we would prevent them from migrating

I agree there shouldn't be a ```NULL``` for ```id``` but I can't prove that it couldn't have happened

Is there a way we can do this without adding this constraint?